### DISCUSSION TITLE
How can you create tags in a repository

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#76951] [Repositories] [answered]
### Select Topic Area

Question

### Body

How can you create tags in a repository?

**Comment by guspan-tanadi** [comment, marked as answer]
Create tags in repository using GitHub for example need a release at first. If prefer to use command line interface please check [Sahil](#discussioncomment-7670433) answer.

From GitHub in the repository click 0 tags link.

![Screenshot from 2023-11-26 08-36-16](https://github.com/community/community/assets/36249910/c11fb55e-23ad-4a6a-9f24-b07e928df49c)

Then Create a new release button.

![Screenshot from 2023-11-26 08-36-45](https://github.com/community/community/assets/36249910/c95b2163-5515-48a2-9d2d-9d7aaa350094)

Next will have appear a form as below. Click dropdown Choose a tag.

![Screenshot from 2023-11-26 08-55-44](https://github.com/community/community/assets/36249910/4bf0ad89-81f9-4dd3-bf4e-5a72476788e2)

Directly typing the tag name as example v0.1-beta.9 and click + Create new tag

![Screenshot from 2023-11-26 08-51-06](https://github.com/community/community/assets/36249910/4df2a73d-3286-4e52-9c48-ba775a26f05f)

Fill the Release title input in the form. Click Publish release button.

There should be a tag has been created.

![Screenshot from 2023-11-26 08-59-30](https://github.com/community/community/assets/36249910/2f898f64-e804-4648-b1a8-34720a298439)

Also the release marked by tags.

![Screenshot from 2023-11-26 08-58-44](https://github.com/community/community/assets/36249910/8d91a83d-2452-4deb-aab1-e088ce711252)

Other relevant on create tags can be achieved through [GitHub Desktop](https://docs.github.com/en/desktop/managing-commits/managing-tags-in-github-desktop).

**Comment by notsointresting** [comment]
To create and push tags in a Git repository, use the following commands:

```bash
# Navigate to your repository
cd /path/to/your/repository

# Create a tag
git tag -a v1.0 -m "your message here"

# Push the tag to the remote repository
git push origin v1.0
```

Replace `v1.0` with your tag and `"your message here"` with your message.
